payments-ui-core/src/main/java/com/stripe/android/ui/core/IsStripeCardScanAvailable.kt

```diff
@@ -1,10 +1,10 @@
 package com.stripe.android.ui.core
 
-internal interface IsStripeCardScanAvailable {
+internal interface IsHCaptchaAvailable {
     operator fun invoke(): Boolean
 }
 
-internal class DefaultIsStripeCardScanAvailable : IsStripeCardScanAvailable {
```

Comment:
mistake?

Reply:
yep, shouldn't have modified this. thanks